Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -867,15 +874,15 @@ private void ApplyTargetTransform(MixedRealityTransform targetTransform)
 
                 if (!transformUpdated.HasFlag(TransformFlags.Move))
                 {
-                    HostTransform.position = isSmoothing ? Smoothing.SmoothTo(HostTransform.position, targetTransform.Position, moveLerpTime, Time.deltaTime) : targetTransform.Position;
+                    HostTransform.position = isSmoothing ? smoothingLogic.SmoothPosition(HostTransform.position, targetTransform.Position, moveLerpTime, Time.deltaTime) : targetTransform.Position;
```

Comment:
Does `smoothingLogic` need to be null checked, in case the `SystemType` specified doesn't exist or can't otherwise be created?

Reply:
I'm not sure, but seems like a good thing to add in case Activator.CreateInstance() can return null. Nice catch.